pkg/openstack/loadbalancer.go

```diff
@@ -467,7 +468,7 @@ func (lbaas *LbaasV2) createOctaviaLoadBalancer(name, clusterName string, servic
 	}
 
 	if svcConf.supportLBTags {
-		createOpts.Tags = []string{svcConf.lbName}
+		createOpts.Tags = append(createOpts.Tags, lbaas.getCustomLoadBalancerTags(service)...)
```

Comment:
I think `.Tags` is a nil here, you don't need to use `append()`, just assign directly.